client/http_sender.go

```diff
@@ -123,22 +120,20 @@ func (s *httpSender) post(call proto.Call) error {
 
 		resp, err := s.client.Do(req)
 		if err != nil {
-			return retry.Continue, err
+			continue
 		}
-
 		defer resp.Body.Close()
```

Comment:
> in go's http client, failing to close a response body prevents that connection from being reused

Ah, I didn't know this (and was otherwise fine with closing at the very end of the function). I'll fix this up.